CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ breaking changes in the upcoming 3.x release. This release is scheduled for
 
 ## v2.26.0 - TBD
 
+### BREAKING TESTING CHANGES
+If you don't have tests that `EXPECT_CALL` on methods in `MockConnection` that
+perform Long Running Operations (LRO), these changes will not affect you.
+
+With the addition of new methods to support starting Long Running Operations
+(LRO) synchronously and awaiting their completion separately, the overload set
+for the preexisting LRO methods have been expanded. Uses of `EXPECT_CALL` that
+do not have matchers for the arguments will be ambiguous. To quickly remedy this
+change instances of `EXPECT_CALL(*mock, Method)` to
+`EXPECT_CALL(*mock, Method(_))`.
```

Comment:
optional: we wouldn't write this, but maybe `s/Method(_)/Method(::testing::_)/` for those who are not familiar with the placeholder.